### ISSUE TITLE
What is doc?

### ISSUE THREAD

**Issue by Kevin-Chen0** [TeamHG-Memex/eli5#390]
Hi eli5?

I'm looking at explain_prediction_xgboost but am unable to understand the mandatory arg **doc** stand for. Is it suppose to a X_test pandas DataFrame? I do not see it being mentioned in the docs: https://eli5.readthedocs.io/en/latest/autodocs/xgboost.html 

```
def explain_prediction_xgboost(
        xgb, **doc**,
        vec=None,
        top=None,
        top_targets=None,
        target_names=None,
        targets=None,
        feature_names=None,
        feature_re=None,  # type: Pattern[str]
        feature_filter=None,
        vectorized=False,  # type: bool
        is_regression=None,  # type: bool
        missing=None,  # type: bool
        ):
```

**Comment by lopuhin** [comment]
hi @Kevin-Chen0 , ``doc`` means one example (or input) to the classifier - and this method explains prediction for it. In case of pandas dataframe, I hope ``X_test.iloc[i]`` should work but not 100% sure.

**Comment by Kevin-Chen0** [comment]
Can u pass in an entire DataFrame or multiple rows, or does explain_prediction*() only work for one row or observation at a time?

**Comment by lopuhin** [comment]
currently, explain_prediction work only for one row/example at a time. There is also explain_weights which needs only the trained model and gives a summary of overall feature importance.
